.pre-commit-config.yaml

```diff
@@ -32,11 +32,6 @@ repos:
           - id: isort
             args: [--profile, black, --filter-files]
 
-    - repo: https://github.com/asottile/setup-cfg-fmt
-      rev: v2.2.0
-      hooks:
-          - id: setup-cfg-fmt
-
```

Comment:
Why do we remove the `setup.cfg` formatting?

Reply:
It currently doesn't support the 'file: ' syntax.

https://github.com/asottile/setup-cfg-fmt/issues/161

Since we want to move to project.toml anyway I think it's okay to delete it. I looked at what it does and it doesn't seem so useful when the setup.cfg is mostly stable. (and in general I hate when linters become obstacles instead of being helpful). But I am happy to concede this point.

Reply:
I put the pre-commit hook back.